libs/src/services/discoveryProvider/DiscoveryProvider.ts

```diff
@@ -1464,7 +1464,8 @@ export class DiscoveryProvider {
         response
       })) ?? response
 
-    return response as unknown as DiscoveryResponse<Response>
+    const responseBody: DiscoveryResponse<Response> = await response.json()
+    return responseBody
```

Comment:
do other consumers of this care?

Reply:
like care about the type? technically there is a response type threaded through the whole function so keeping the types accurate

Reply:
but json is new right?

Reply:
its cause we using `fetch` in this case, so we have to `.json` here, whereas axios does this automatically!

Reply:
no, it's not new - this is from the forked version of DiscoveryProvider's `_makeRequest` that Dylan created for the new DN selector logic. The old path used Axios which auto parses the JSON response. This is to match feature parity with the old way

Reply:
look at us :)

Reply:
👍